t/t1020-mf-priority-issue262.t

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+test_description='Test comparing job counts when submitting jobs that take up all resources'
```

Comment:
Nicely done. Did you verify that before the changes in this PR this test fails? (just curious)

Reply:
Yes I did - here are the results of running this test with the plugin before any changes:

#### query_1.json

##### expected
cur_running_jobs: 1
cur_active_jobs: 3

##### failed
cur_running_jobs: 3
cur_active_jobs: 3

---

#### query_2.json

##### expected
cur_running_jobs: 1
cur_active_jobs: 3

##### failed
cur_running_jobs: 5
cur_active_jobs: 3

---

#### query_3.json

##### expected
cur_running_jobs: 1
cur_active_jobs: 3

##### failed
cur_running_jobs: 8
cur_active_jobs: 3

---

#### query_4.json

##### expected
cur_running_jobs: 1
cur_active_jobs: 2

##### failed
cur_running_jobs: 7
cur_active_jobs: 2

---

#### query_5.json

##### expected
cur_running_jobs: 0
cur_active_jobs: 0

##### failed
cur_running_jobs: 5
cur_active_jobs: 0